src/css/_layout.scss

```diff
@@ -76,14 +76,14 @@
 .guide {
 	font-family: raleway light, sans-serif;
 	font-size: 20px;
-	color:#f8f9eb;
+	color: #f8f9eb;
 	text-align: center;
 }
 
-.box1{
-	border-radius: 0px;
-	background-size: cover;
+.box1 {
+	border-radius: 0;
 	background: url("https://www.bostonmagazine.com/wp-content/uploads/sites/2/2019/10/Greenway-NorthEnd-main.jpg");
```

Comment:
Make sure this is in your images folder - if you rely on bostonmagazine.com to host this, it could break at any time!